docs/guide/deployer.md

```diff
@@ -168,6 +168,55 @@ For opting in to App, `deployer` supports the following methods:
 You can learn more about Stateless Smart Contracts [here](https://developer.algorand.org/docs/features/asc1/stateless/).
 
 
+#### Checkpoint names
+
+Storing data in [checkpoint](https://algobuilder.dev/guide/execution-checkpoints.html) against your own names can be very useful. For example, you don't need to pass smart contract template parameters every time while getting app info, or loading a logic signature. Currently, we support naming for apps (Algorand stateful application) and logic signatures.
```

Comment:
```suggestion
Algob creates [checkpoint](https://algobuilder.dev/guide/execution-checkpoints.html) and associates them with a name. This is a very useful feature. For example, you don't need to pass smart contract template parameters every time when getting app info, or loading a logic signature. Since `algob v4.0`, we support naming for apps (Algorand stateful smart contracts) and smart signatures.
```